packages/server/eslint.config.mjs

```diff
@@ -38,6 +38,12 @@ const configs = [
       }
     },
     rules: {
+      'no-restricted-imports': [
+        'error',
+        {
+          patterns: ['.*']
```

Comment:
How's this work? Doesn't a `.*` regex resolve to literally any kind of string?

Reply:
Ah, these can be glob patterns too apparently. Well, in any case, there's some paths that need fixing picked up by pre-commit

Reply:
@iainsproat That's alright, we can add an eslint-disable comment there, cause that's an exception scenario.